css/bootstrap.css

```diff
@@ -3569,7 +3569,7 @@ select[multiple].input-group-sm > .input-group-btn > .btn {
 .nav-pills > li.active > a:hover,
 .nav-pills > li.active > a:focus {
   color: #ffffff;
-  background-color: #428bca;
+  background-color: #b3c2cf;
```

Comment:
Don't change bootstrap.css. It comes from Bootstrap directly. You can overwrite stuff in bbgm.css.